mwdb/resources/oauth.py

```diff
@@ -109,8 +116,23 @@ def post(self):
             userinfo_endpoint=obj["userinfo_endpoint"],
             jwks_endpoint=jwks_endpoint,
         )
+
+        group_name = "OpenID_" + obj["name"]
```

Comment:
Group name should match `^[A-Za-z0-9_-]{1,32}$`.
- if name is longer than 32-7=25, it will fail
- if name contains other characters (e.g. space), it will fail as well.

In addition, some users are already using that feature. Maybe we should drop invalid characters from group name or replace them with some character? (e.g. `_`)

```suggestion
        group_name = ("OpenID_" + obj["name"])[:32]
```